examples/cucumber/.sauce/config.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1alpha
+kind: espresso
+sauce:
+  region: us-west-1
+  concurrency: 10 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Cucumber and Espresso Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+espresso:
+  app: ./apps/espresso-cucumber.apk
+  testApp: ./apps/espresso-cucumber-success.apk
+suites:
+  - name: "saucy barista"
+    devices:
+      - name: "Android GoogleApi Emulator"
+        orientation: portrait
```

Comment:
@valentyn88 can you remove the `orientation` we didnt check if it's working yet.

Reply:
I removed it.